### PR TITLE
Add configuration to support API

### PR DESCRIPTION
**Pull request by anandcsingh**
## How to run
- navigate to api folder
- npm install
- npm run build
- npm run start
- Invoke API using Postman or the like 
- URL like http://localhost:3000/api/credentials
## Sample json
`{
    "name": "License",
    "fields":[
        { "name": "number", "type": "CircuitString"},
        { "name": "name", "type": "CircuitString"}
    ]
}`

## Details of change
Made some minor changes to configure the api
- Add the build command to package.json
- changed the tsconfig
- included the .js file extension in imports
- added a res.send() call at the end of the controller to complete the request

### CODE REVIEW

File: api/src/app.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import express from 'express';
-import * as bodyParser from 'body-parser';
-import { credsRouter } from './routes/credentialsRoute';
+import bodyParser from 'body-parser'
+import { credsRouter } from './routes/credentialsRoute.js';
```
Comment (kerisha) [inline api/src/app.ts:3]: why .js extensions for imports were added to .ts files?

Comment (kerisha) [reply api/src/app.ts:3]: are there any implications for doing such a thing?

**Comment by kerisha** [comment]
It works but I do not understand the file extensions with referencing... no .js file exists outside of the dist folder. so what is it referencing?

File: api/src/app.ts (hunk 1 above)
Comment (anandcsingh) [reply api/src/app.ts:3]: Well there is likely some issue with the typescript configuration, but everything compiles down to JavaScript so you should be fine.
Don't think much side effects other than lack of consistency, I have mixed both in other projects

**Comment by anandcsingh** [comment]
> It works but I do not understand the file extensions with referencing... no .js file exists outside of the dist folder. so what is it referencing?

The dist folder is like the bin folder in c# when you run these are the actual files that get run. Everytime you run build you invoking the typescript compiles to emit JavaScript to the output folder in this case dist

File: api/src/app.ts (hunk 1 above)
Comment (kerisha) [reply api/src/app.ts:3]: ok I don't like it. lol but it works.... and I do not know an alternative.

**Pull request merged**